content/docs/install/helm-install/index.md

```diff
@@ -65,29 +65,43 @@ To install the Control Plane, you must do the following:
 
 ### Simple Keptn installation
 
-  The following commands provide a basic Keptn installation.
-  Watch a video demonstration of this simple installation [here](https://www.youtube.com/watch?v=neAqh4fAz-k).*
-   ```
-   helm repo add keptn https://charts.keptn.sh
-   helm install keptn keptn/keptn -n keptn --create-namespace
-   kubectl -n keptn port-forward svc/api-gateway-nginx 8080:8080
-   ```
+The following commands provide a basic Keptn installation
+This set of commands is useful for creating a basic Keptn installation
+to use for study or demonstration.
+It is not adequate for a production Keptn installation.
+
+Watch a video demonstration of this simple installation
+[here](https://www.youtube.com/watch?v=neAqh4fAz-k).
+
+The first command sequence installs the Keptn control plane
+and exposes it using an [Ingress](../access/#option-3-expose-keptn-via-an-ingress):
 
-  We use **kubectl** to forward port `8080` from our local machine
-  to port `8080` on the Keptn API Gateway service in the cluster.
+```
+helm repo add keptn https://charts.keptn.sh
+helm install keptn keptn/keptn -n keptn --create-namespace
+kubectl -n keptn port-forward svc/api-gateway-nginx 8080:8080
+```
+We use **kubectl** to forward port `8080` from our local machine
+to port `8080` on the Keptn API Gateway service in the cluster.
 
-  This set of commands is useful for creating a basic Keptn installation
-  to use for study or demonstration.
-  It is not adequate for a production Keptn installation.
+Here is an alternate command for installing a basic Keptn installation,
+this time using a [LoadBalancer](../access/#option-1-expose-keptn-via-a-loadbalancer) to expose Keptn:
 
+```
+helm install keptn keptn --repo=https://charts.keptn.sh \
+-n keptn --create-namespace \
+--set=apiGatewayNginx.type=LoadBalancer \
+--set=continuousDelivery.enabled=true \
```

Comment:
The `continuousDelivery.enabled` setting is gone since the removal of helm-service from the main repo. I guess this file should represent the latest version since it's not under a version folder, so I think this needs to be removed.

Reply:
done